src/js/services/lang.js

```diff
@@ -21,25 +21,20 @@ const languages = {
   },
 };
 
-// Sets the new url for the lanuage selected
-const redirectLocation = (langCode) => {
-  const parser = new RegExp(
-    '^(https?://' + window.location.host + '/)(docs/[a-z]{2}/|docs/)(.*)',
-    's',
-  );
-  const baseURL = 'https://circleci.com/' + languages[langCode].url + '2.0';
```

Comment:
Wooo! So happy got it to work without the url hardcoded! Very nice job on this!!